src/main/java/org/apache/commons/cli/CommandLine.java

```diff
@@ -691,6 +692,203 @@ public <T> T getParsedOptionValue(final String opt, final T defaultValue) throws
         return getParsedOptionValue(resolveOption(opt), defaultValue);
     }
 
+    /**
+     * Gets a version of this {@code Option} converted to an array of a particular type.
+     *
```

Comment:
Can the method return null? If so, when? What happens if the option does not exist?

Reply:
OK, then please update the Javadoc instead of writing it up here ;-)